src/components/feedback-form.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+// import Form from "./form";
+import styles from "./call-to-action.module.css";
+import PropTypes from "prop-types";
+import Form from "./form";
+
+
+function FeedbackForm() {
+
+  const additionalFormAttr = {
+    "data-netlify": "true",
+    method: "post"
+  }
+  return (
+    <Form
+      className={`flex flex-col rounded shadow w-full mb-4 lg:w-1/2 lg:mr-4 p-6`}
```

Comment:
We can remove the `lg:mr-4` selector. It's making the form look a bit off center.

```suggestion
      className={`flex flex-col rounded shadow w-full mb-4  p-6 lg:w-1/2`}
```